admin-dev/themes/new-theme/js/pages/module/controller.js

```diff
@@ -390,7 +431,7 @@ class AdminModuleController {
 
     const counter = {};
 
-    for (let i = 0; i < self.modulesList.length; i++) {
+    for (let i = 0; i < self.modulesList.length; i += 1) {
```

Comment:
Could you store the value of `self.modulesList.length` outside? It is not cached and calculated at each loop.

Reply:
Done